context.go

```diff
@@ -6,6 +6,11 @@ type Context struct {
 	Params
 }
 
+// Init initializes data so that context can be set
+func (r *Context) Init() {
```

Comment:
Let's change this to something like MakeContext()
And initialize a new Context object with data set to `make(map[string]interface{})`